packages/commonwealth/client/scripts/controllers/chain/substrate/shared.ts

```diff
@@ -254,15 +258,7 @@ class SubstrateChain implements IChainModule<SubstrateCoin, SubstrateAccount> {
 
   // load existing events and subscribe to future via client node connection
   public async initChainEntities(): Promise<void> {
-    await this._app.chainEntities.refresh(this.app.chain.id);
-    const subscriber = new SubstrateEvents.Subscriber(this.api);
-    const processor = new SubstrateEvents.Processor(this.api);
-    return this._app.chainEntities.subscribeEntities(
-      this._app.chain.id,
-      chainToEventNetwork(this.app.chain.meta),
-      subscriber,
-      processor
-    );
+    console.warn('Substrate chain-events are not supported!');
```

Comment:
Do we still get active proposals directly from chain? Not a big deal if we don't, but want to know.

Reply:
We haven't for some time: https://github.com/hicommonwealth/commonwealth/pull/2091